iputils/README.md

```diff
@@ -0,0 +1,24 @@
+# Iputils Dynamic Build
+The **iputils** package is a set of small utilities for Linux networking.
+This directory includes the dynamic application build, that includes, the
+binary as well as the library dependencies for running inside of Unikraft.
```

Comment:
Add wording as in the [`README.md` file for `lang/lua`](https://github.com/unikraft/dynamic-apps/blob/master/lang/lua/README.md#lua-as-dynamic-pie-executable).

Reply:
Do a listing of the executables and their purpose.